docs/mobile/exercises/select-table.md

```diff
@@ -0,0 +1,37 @@
+# Select Table
+
+A Select Table exercise is essentially identical to a [Select Output
+exercise](select-output.md), but instead of a mandatory `output` block, there is
+a mandatory `table` block. This is ideal for exercises when the output of a code
+block is better represented as a table - SQL queries, for example.
+
+```yaml
+- key: 680a2682-4177-4d87-a404-c8044153d57f
+  code: "SELECT * FROM team_mobile"
+  table:
+    data: |-
+      Name   ,Awesomeness,Role
```

Comment:
Is this fixed width format, or CSV with extra whitespace for human readability? Best to clarify this.

Reply:
I explain this in the high level exercise page, but it would be good to mention it again here.